save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -63,7 +58,7 @@ class FixPlugin(
                 } catch (ex: ProcessExecutionException) {
                     return@map TestResult(
                         listOf(expected, test),
-                        Fail(ex.message!!),
+                        Fail("${ex::class.simpleName}: ${ex.message}", ex::class.simpleName ?: "Unknown exception"),
```

Comment:
What's the point to use just the class name of exception as `shortReason`? Also `ProcessExecutionException` is not anonymous object, so I think `simpleName` always exists

Reply:
Yes, this makes sense. Actually, I'm surprised, that pure Kotlin doesn't have Java-style toString for Throwables, with class and message. I've added it in https://github.com/cqfn/save/pull/154/commits/934e5ea9c41f6cec319ec51c117b41efab8c7175